book/website/collaboration/organising-conference.md

```diff
@@ -307,5 +307,5 @@ Provide plenty of breaks, and treat those breaks as virtual coffee sessions.*
 
 ## Resources
 
-- The *Checklist for running a meeting* is a summarised version of the [Gallantries (Galaxy+The Carpentries handbook)](https://galaxy-carpentries.github.io/gallantries/handbook) which is an expanded version of [The Carpentries workshop checklist](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html).
+- The *Checklist for running a meeting* is a summarised version of the [Gallantries (Galaxy+The Carpentries handbook)](https://gallantries.github.io/) which is an expanded version of [The Carpentries workshop checklist](https://docs.carpentries.org/).
```

Comment:
```suggestion
- The *Checklist for running a meeting* is a summarised version of the [Gallantries (Galaxy+The Carpentries handbook)](https://gallantries.github.io/handbook#workshop-checklists) which is an expanded version of [The Carpentries workshop checklist](https://docs.carpentries.org/resources/workshops/checklists.html).
```
You found the right pages! I'd suggest we link directly to the checklists we're referring to on each site which is what this update does.